src/Container/InMemoryEventStoreFactory.php

```diff
@@ -198,8 +197,7 @@ private function createActionEventEmitterDecorator(
         ActionEventEmitter $actionEventEmitter
     ): ActionEventEmitterEventStore {
         if ($this->isTransactional) {
-            assert($eventStore instanceof TransactionalEventStore);
-
+            /** @var TransactionalActionEventEmitterEventStore $eventStore */
```

Comment:
TransactionalActionEventEmitterEventStore is wrong, it's TransactionalEventStore, but you can also remove this line

Reply:
Ah sorry, did that in hurry. Without that line IDE shines with error (and autosuggest does not work) which I don't like... Is this line problem or can I leave it after fixing classname?

Reply:
sure, leave but fix class name